### PR TITLE
Add @pjanotti as codeowner for Windows perf counter and related components

### PR DESCRIPTION
**Pull request by pjanotti**
Adding myself as a codeowner for Windows performance counter and related components per respective call for more volunteers on the respective README files.

I have experience with Windows and was a maintainer of the collector some years ago.

Per DM I am also removing @djaglowski from `receiver\windowseventlog`, ActiveDirectoryDS, and IIS.

cc @alxbl

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -197,7 +197,7 @@ processor/sumologicprocessor/                                       @open-teleme
 processor/tailsamplingprocessor/                                    @open-telemetry/collector-contrib-approvers @jpkrohling
 processor/transformprocessor/                                       @open-telemetry/collector-contrib-approvers @TylerHelmuth @kentquirk @bogdandrutu @evan-bradley
 
-receiver/activedirectorydsreceiver/                                 @open-telemetry/collector-contrib-approvers @djaglowski @BinaryFissionGames
+receiver/activedirectorydsreceiver/                                 @open-telemetry/collector-contrib-approvers @djaglowski @BinaryFissionGames @pjanotti
```
Comment (pjanotti) [inline .github/CODEOWNERS:200]: @djaglowski would like to be removed from this one and the IIS one?

Comment (djaglowski) [reply .github/CODEOWNERS:200]: Yes, I'll happily drop from those. Thank you for taking them on.

Comment (pjanotti) [reply .github/CODEOWNERS]: [1f5b222](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33695/commits/1f5b222295ed89af3ebb3fffa86924771629a885)

**Review by djaglowski — APPROVED** [review]

**Comment by dmitryax** [comment]
@pjanotti Can you please update the metatada.yaml files accordingly as well https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/windowsperfcountersreceiver/metadata.yaml#L10?

**Review by alxbl — APPROVED** [review]

**Comment by crobert-1** [comment]
CI/CD failure:
```
Generated code is out of date, please run "make generate" and commit the changes in this PR.
```
_This will update relevant READMEs to show you're a code owner._

**Review by crobert-1 — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**